photon-server/src/main/java/org/photonvision/vision/frame/consumer/MJPGFrameFileSaveConsumer.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2020 Photon Vision.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.photonvision.vision.frame.consumer;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+import java.io.File;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.opencv.imgcodecs.Imgcodecs;
+import org.photonvision.common.configuration.ConfigManager;
+import org.photonvision.common.logging.LogGroup;
+import org.photonvision.common.logging.Logger;
+import org.photonvision.vision.frame.Frame;
+
+public class MJPGFrameFileSaveConsumer {
+
+    private static String FILE_PATH = ConfigManager.getInstance().getImageSavePath().toString();
+    private static String FILE_EXTENSION = ".jpg";
+    DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
+    DateFormat tf = new SimpleDateFormat("hhmmssSS");
+
+    private final String ntEntryName = "saveImgCmd";
+
+    private final Logger logger;
+
+    private final NetworkTable table;
+
+    private boolean prevCommand = false;
+
+    private String sourceName;
+
+    public MJPGFrameFileSaveConsumer(String sourceName) {
+        this.sourceName = sourceName;
+        
+        this.table = NetworkTableInstance.getDefault().getTable("/CameraFrameSaver").getSubTable(this.sourceName);
```

Comment:
I wonder if this entry might better belong to each camera, since each vision module holds two of these. So like /photonvision/gloworm/saveImgCmd or something. Note that we'll have to add this to photonlib too.

Reply:
Yup, I can move that. My NT location choice was arbitrary and only designed to be "out of the way".  

Will move into the camera and condense all MJPGFrameFileSaveConsumer's to the referenced location. Any preference on whether we'd want one command per stream (input/output)? Or a single command for all?